src/components/generic_flow_page.tsx

```diff
@@ -12,13 +9,17 @@ export interface GenericFlowPageProps {
     description?: string;
     image?: JSX.Element;
     children?: any;
+    appName?: string;
 }
 
 export const GenericFlowPage: React.StatelessComponent<GenericFlowPageProps> = ({
-    title, description, image, children,
+    title, description, image, children, appName,
 }) => (
     <div className="generic-flow-page">
-        <div className="generic-flow-page__logo"><UltimakerLogo /></div>
+        <div className="generic-flow-page__header">
+            <span className="generic-flow-page__um-text">Ultimaker</span>
+            <span className="generic-flow-page__app__name show-sm">{appName}</span>
```

Comment:
Since appName is optional, this span should only be conditionally rendered when appName is actually passed.

Reply:
The condition is now added; <span>...</span> will only be rendered when an `appName` exists:
https://github.com/Ultimaker/react-web-components/pull/239/commits/04d49a16af414e268a31fe35a7daf3db36f9a4ed